.bcr/config.yml

```diff
@@ -0,0 +1,3 @@
+fixedReleaser:
+  login: jsharpe
+  email: james.sharpe@zenotech.com
```

Comment:
We may be able to remove this if bazel-contrib/bazel-central-registry gets the publish-to-bcr app configured on it.

Reply:
App should be configured for all repos in bazel-contrib so this should work ok without a fixed releaser.